test/livebook/runtime/evaluator_test.exs

```diff
@@ -1300,6 +1300,75 @@ defmodule Livebook.Runtime.EvaluatorTest do
       assert_receive {:runtime_evaluation_response, :code_1, terminal_text("6"), metadata()}
     end
 
+    test "evaluate erlang-module code", %{evaluator: evaluator} do
+      Evaluator.evaluate_code(
+        evaluator,
+        :erlang,
+        "-module(tryme). -export([go/0]). go() ->{ok,went}.",
+        :code_4,
+        []
+      )
+
+      assert_receive {:runtime_evaluation_response, :code_4, terminal_text(_), metadata()}
+    end
+
+    test "evaluate erlang-module error function already defined", %{evaluator: evaluator} do
+      Evaluator.evaluate_code(
+        evaluator,
+        :erlang,
+        "-module(tryme). -export([go/0]). go() ->{ok,went}. go() ->{ok,went}.",
+        :code_4,
+        []
+      )
+
+      assert_receive {
+        :runtime_evaluation_response,
+        :code_4,
+        error(message),
+        metadata()
+      }
```

Comment:
We can help the formatter a bit:

```suggestion
      code = """
      -module(tryme).
      -export([go/0]).

      go() -> {ok,went}.
      go() -> {ok,went}.
      """

      Evaluator.evaluate_code(evaluator, :erlang, code, :code_1, [])

      assert_receive {:runtime_evaluation_response, :code_1, error(message), metadata()}
```

(same in other places)

Reply:
Cleanup in last commit.